stl/inc/ranges

```diff
@@ -260,52 +260,55 @@ namespace ranges {
     template <bool _Enable, class _Rng, class _Derived>
     using _Cached_position_t = conditional_t<_Enable, _Cached_position<_Rng, _Derived>, view_interface<_Derived>>;
 
-    template <_Copy_constructible_object _Ty>
-    class _Copyable_box { // a simplified optional that augments copy_constructible types with full copyability
+    // a simplified optional that augments copy_constructible types with full copyability,
+    // and move_constructible types with full moveability.
+    template <_Move_constructible_object _Ty>
+        requires is_object_v<_Ty>

```

Comment:
This line is problematic and should be removed. `_Move_constructible_object` is already requiring `_Ty` to be an object type.
```suggestion
```